src/server/system_services/schemas/nsfs_config_schema.js

```diff
@@ -144,8 +146,9 @@ module.exports = {
         host_customization: {
             type: 'object',
             patternProperties: {
-                '^[a-zA-Z0-9]$': { $ref: '#/definitions/nsfs_node_config_schema' }
+                '\\w+': { $ref: '#/definitions/nsfs_node_config_schema' }
             },
+            additionalProperties: false,
```

Comment:
remove patternProperties and use schema for additionalProperties:
```suggestion
            additionalProperties: { $ref: '#/definitions/nsfs_node_config_schema' },
```